client/my-sites/patterns/components/section/index.tsx

```diff
@@ -13,13 +16,18 @@ type PatternsSectionProps = {
 
 export const PatternsSection = ( {
 	title,
+	forwardRef,
+	id,
 	description,
 	theme,
 	bodyFullWidth,
 	children,
 }: PatternsSectionProps ) => {
+	const sectionProps = id ? { id } : {};
 	return (
 		<section
+			ref={ forwardRef }
+			{ ...sectionProps }
```

Comment:
The previous discussion about this was solved, but didn't get any answer to my last comment: https://github.com/Automattic/wp-calypso/pull/92794#discussion_r1684502642